rpcd/playbooks/roles/logstash/tasks/upgrade_logstash_pre.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Remove existing logstash
+  apt:
+    package: logstash
+    state: absent
```

Comment:
We may need a `purge: yes` here, to ensure the old /etc/init/logstash* scripts get removed.